beeline.go

```diff
@@ -179,6 +193,15 @@ func Init(config Config) {
 	if config.PresendHook != nil {
 		trace.GlobalConfig.PresendHook = config.PresendHook
 	}
+
+	if config.TraceHTTPHeaderParserHook != nil {
```

Comment:
```suggestion
	if len(config.TraceHTTPHeaderParserHook) > 0 {
```

Reply:
Not sure I understand this suggestion. `config.TraceHTTPHeaderParserHook` is a function, so it's zero-value is `nil`, so this is the correct check, right?

Reply:
Oh, I'm sorry. I mistook these two fields for the slices I saw elsewhere, thinking that you were checking if a slice field was nil. I rescind my suggestion here and below.